### PR TITLE
Feature/configure drive talon

### PR DESCRIPTION
**Pull request by johnli01**
Added Brake/Coast and VoltageLimit for master and follower

### CODE REVIEW

File: Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java [hunk 1]
```diff
@@ -99,6 +101,19 @@ private void configureMotorTeam(
         master.createTelemetryProperties(masterName);
         
         this.updateMotorPidValues(master);
+        
+        // Master Config
+        master.setNeutralMode(NeutralMode.Coast);
```
Comment (WasabiFan) [inline Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java]: Hmmm... does this need to be done for the follower as well?

**Review by WasabiFan — APPROVED** [review]

File: Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java (hunk 1 above)
Comment (johnli01) [reply Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java]: Yeah it'd be safe to add that in

File: Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java [hunk 2]
```diff
@@ -51,56 +55,59 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager, El
...
         master.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 0);
         master.setSensorPhase(sensorPhase);
         master.createTelemetryProperties(masterName);
-        
+
         this.updateMotorPidValues(master);
+
+        // Master Config
+        master.setNeutralMode(NeutralMode.Coast);
+        master.configForwardLimitSwitchSource(LimitSwitchSource.Deactivated, LimitSwitchNormal.Disabled, 0);
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java:98]: You also need to `configReverseLimitSwitchSource`

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java [hunk 3]
```diff
@@ -51,56 +55,60 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager, El
...
+        master.configReverseLimitSwitchSource(LimitSwitchSource.Deactivated, LimitSwitchNormal.Disabled, 0);
```
Comment (WasabiFan) [inline Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java:99]: What motivated replacing `overrideLimitSwitchesEnable` with `configReverseLimitSwitchSource`?

Comment (johnli01) [reply Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java:99]: If I recall it was something about override that seemed mildly discomforting

Comment (JohnGilb) [reply Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java:99]: After reading some docs, this is the best method for 100% disabling any limit switch read on a talon.

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**